app/main.py

```diff
@@ -2,0 +1,15 @@
+def copy_file(command: str) -> None:
+    split = command.split(" ")
+    command = split[0]
+
+    if command != "cp":
+        return
+
+    file1 = split[1]
```

Comment:
The variable name `file1` is not descriptive. Consider renaming it to something like `source_file` to better convey its purpose.

Reply:
agreed